### PR TITLE
[PATCH v5] Two fxes for sysinfo

### PR DESCRIPTION
**Pull request by lumag**

### CODE REVIEW

File: platform/linux-generic/odp_system_info.c [hunk 1]
```diff
@@ -443,7 +443,7 @@ int odp_sys_huge_page_size_all(uint64_t size[], int num)
 	dir = opendir("/sys/kernel/mm/hugepages");
 	if (!dir) {
 		ODP_ERR("Failed to open huge page directory\n");
```
Comment (muvarov) [inline platform/linux-generic/odp_system_info.c]: ODP_ERR -> ODP_DBG then.

Comment (lumag) [reply platform/linux-generic/odp_system_info.c]: Why? It's still an error, but just not critical error.

File: platform/linux-generic/arch/x86/odp_sysinfo_parse.c [hunk 2]
```diff
@@ -15,34 +15,54 @@ int cpuinfo_parser(FILE *file, system_info_t *sysinfo)
...
+				freq_set = true;
+			}
+		} else if (!freq_set &&
+			   strstr(str, "bogomips") != NULL) {
+			pos     = strchr(str, ':');
+			if (pos == NULL)
+				continue;
+
+			if (sscanf(pos + 2, "%lf", &mhz) == 1) {
+				/* On typical x86 BogoMIPS is freq * 2 */
```
Comment (muvarov) [inline platform/linux-generic/arch/x86/odp_sysinfo_parse.c]: First 2 patches are ok. But I'm not sure about that statement. My i7-4710HQ CPU @ 2.50GHz has 4988.28 bogomips. Which is about double. But frequency jumps from 1Ghz to 2.5Ghz. If we say odp app rolls in busy loop than it looks like true. At least it's better to set up some dummy number. Other solution might be to calculate frequency with odp_cpu_cycles() (rdtsc) and clock_gettime().

**Comment by Bill-Fischofer-Linaro** [comment]
Typo in the PR name (fixes, not fxes). I assume this can be fixed on merge?

**Comment by muvarov** [comment]
@Bill-Fischofer-Linaro it's not typo in patch itself. Nothing to correct.

File: platform/linux-generic/arch/x86/odp_sysinfo_parse.c (hunk 2 above)
Comment (lumag) [reply platform/linux-generic/arch/x86/odp_sysinfo_parse.c]: It's about max frequency, of course.

**Comment by muvarov** [comment]
Looks like this can be merged now.

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**